### DISCUSSION TITLE
Transmission is still not working

### DISCUSSION THREAD

**Discussion by albabe** [transmission/transmission#5330] [General]
4.00 completely destroyed all my torrents, either just eliminating them, showing unfinished and not allowing me to recheck them... and with .01 and .02 it rearranges all my torrents every time I restart the App or my Computer.

**Comment by albabe** [comment]
I realize you folks are busy, but could someone tell me why every time I start the App, Transmission rearranges  my torrents.  And not in Alphabetical order either...

**Reply by ile6695** [reply]
Answering would be easier if you tell more.
- What OS you have?
- Do you use GTK, QT5, QT6 or Web version of Transmission? For windows 10 it's likely QT6.
- Is the rearranged order always the same? It could be sorted by some other way. For example size.

**Reply by albabe** [reply]
So... any progress?

**Comment by albabe** [comment]
Oops... sorry.  My Bad

Monterey 12.4
Dunno what those are.  I got it from your site for Mac.
The rearrange order seems the same each time.
My Pull Down:  View > Sort Transfers by > Que Order
I'm using Groups

Any idea?

**Comment by ckerr** [comment]
> I realize you folks are busy, but could someone tell me why every time I start the App, Transmission rearranges my torrents. And not in Alphabetical order either...

Duplicate of https://github.com/transmission/transmission/issues/155 ?

**Reply by albabe** [reply]
I have no idea what that is... got me.

**Reply by dantearmok** [reply]
yup

**Comment by dantearmok** [comment]
I had no idea that torrents were draggable, when you had Sort by Queue Order selected, until I read this ticket! LOL

Yes, we need to fix this.  The state(order) should be saved between sessions.

IMO, it's not necessarily an intuitive/useful part of the interface, tho.  I think it's more important is to balance 
"Prefs> Global maximum connections", 
"Prefs> Maximum connections for new transfers", 
and the Bandwidth rates.

eg: I have 60/20 Mbps (6000/2000 KB/sec) usable bandwidth and a reasonably fast CPU and HDD.  So I set it to 32 connections per torrent and 1024 max connections.  Then I control Transmission's behavior with the Bandwidth settings.  During the day, I turtle it down to something that those around me don't mind (1000/500 KB/sec), and during the night, when we unturtle, I set the download and upload rates to about 80% of the usable bandwidth.

hth,
- Dan.

**Reply by albabe** [reply]
By "Draggable," I assume you mean Drag to a different order and drop?  I apologize for my ignorance.

"Que Order" would be correct, right?

Do you know if this happening with all Platforms or just in Monterey?

I have so many Torrents.  I almost always keep them seeding for weeks or months or even years.  In fact, I have Torrents for Comic Books I wrote and Drew that I've been seeding for decades.  "Que Order" sure helps me organize.

**Reply by dantearmok** [reply]
yes.  When sort is by queue order, you can drag the torrents.  But it looks like this is not working correctly.  I dunno about all platforms, but it's not based on a specific version of macOS.

What about turning on Groups and sorting by Name?  You can add more groups,...

**Reply by albabe** [reply]
I have so many torrents, that would be insane.  It's hard to believe i'm the only one out there that uses "Que Order."  So, I guess no one can fix this?

**Comment by albabe** [comment]
I just tried 4.0.3 and it's still broke.  Anyone?

**Reply by dantearmok** [reply]
The Transmission team is aware of this bug.

See ticket #155 "save queue order between sessions".

Hopefully, someone will work on it soon.

**Comment by ckerr** [comment]
Locking as duplicate of #155.

When reporting an issue, please use the issue tracker, and please check to see if there's already an issue filed, and please use a title more descriptive than "still not working" so that other people also can find the right issue when searching :smile_cat:
